src/network/taraxa_capability.cpp

```diff
@@ -347,6 +365,13 @@ void TaraxaCapability::interpretCapabilityPacketImpl(NodeID const &_nodeID, unsi
           LOG(log_dg_dag_prp_) << "Received NewBlock " << block.getHash().toString() << "that is already known";
           break;
         }
+        if (auto status = checkDagBlockValidation(block); !status.first) {
+          LOG(log_wr_dag_prp_) << "Received NewBlock " << block.getHash().toString()
+                               << "but we are dropping it as we are missing pivot or/and tips";
```

Comment:
missing white space after hash and lets keep messages as short as possible, why not just write "Received invalid dag block - missing pivot and/or tips" ? All this part "but we are dropping it as we are" does not add any meaning/value to the message

Reply:
Ok, I will change this